### PR TITLE
View/iu video

### PR DESCRIPTION
**Pull request by raulchoque-eng**
The UI for search a video was implement in the MainView class

### CODE REVIEW

File: src/main/java/com/jala/view/JPanelAdvanced.java [hunk 1]
```diff
@@ -49,6 +49,22 @@
...
+     */
+    public JLabel getLblExtension() {
+        return lblExtension;
+    }
+
+    /**
+     * The getTxfExtension method get the data txtExtension
+     * @return a component JTextField type
+     */
+    public JTextField getTxfExtension() {
```
Comment (melvicaballero) [inline src/main/java/com/jala/view/JPanelAdvanced.java]: public JTextField getTxfExtension() { change for public JTextField getTxtExtension() {

File: src/main/java/com/jala/view/JPanelSearchGral.java [hunk 2]
```diff
@@ -104,11 +104,14 @@ private void initComponent() {
         JPanel pnlNorth = pnlCriteria();
         this.add(pnlNorth, BorderLayout.NORTH);
 
-        JPanel pnlCentral = pnlButtonAdvanced();
+        /*JPanel pnlCentral = pnlButtonAdvanced();
```
Comment (Regis18) [inline src/main/java/com/jala/view/JPanelSearchGral.java]: Delete this comments

Comment (melvicaballero) [inline src/main/java/com/jala/view/JPanelSearchGral.java]: no code comment

File: src/main/java/com/jala/view/JPanelSearchVideo.java [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+    }
+
+    /**
+     * This method add items in combo box
+     * @param myItems are data for add at cbFrameRat
+     * @param cbFramRat is a data where add items
+     */
+    private void addItemsComboBox(String[] myItems, JComboBox cbFramRat) {
+        for (String item : myItems) {
+            cbFramRat.addItem(item);
```
Comment (melvicaballero) [inline src/main/java/com/jala/view/JPanelSearchVideo.java]: cmbFrameRate change

File: src/main/java/com/jala/view/JPanelSearchAdvancVideo.java [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+import javax.swing.border.TitledBorder;
+import java.awt.BorderLayout;
+
+/**
+ * JPanelSearchAdvancVideo class
+ *
+ * @author Raúl Choque
+ * @version 0.0.1
+ */
+public class JPanelSearchAdvancVideo extends JPanel{
```
Comment (Regis18) [inline src/main/java/com/jala/view/JPanelSearchAdvancVideo.java]: Rewrite name of class

File: src/main/java/com/jala/view/JPanelSearchAdvancVideo.java [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+    public JPanelSearchAdvancVideo() {
+        initComponent();
+    }
+
+    /**
+     * The initComponent method will initialize the components.
+     */
+    private void initComponent() {
+        setLayout(new BorderLayout());
+        panelAdvancSearch = new JPanelAdvanced(new BorderLayout());
```
Comment (Regis18) [inline src/main/java/com/jala/view/JPanelSearchAdvancVideo.java]: change Name of panelAdvancSearch

File: src/main/java/com/jala/view/JPanelSearchVideo.java [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+    }
+
+    /**
+     * Section of getter methods for get String values
+     */
+    public String getCmbFrameRate() {
+        return cmbFrameRate.getSelectedItem().toString();
+    }
+
+    public String getCmbVideoCodec() {
```
Comment (Regis18) [inline src/main/java/com/jala/view/JPanelSearchVideo.java:162]: Add comments

File: src/main/java/com/jala/view/MainView.java [hunk 7]
```diff
@@ -45,7 +47,7 @@
      * Constructor.
```
Comment (Regis18) [inline src/main/java/com/jala/view/MainView.java:47]: Comment Constructor, more details

File: src/main/java/com/jala/view/JPanelSearchAdvancedVideo.java [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+import java.awt.BorderLayout;
+
+/**
+ * JPanelSearchAdvancedVideo class is the panel to do
+ * the search advanced video
+ *
+ * @author Raúl Choque
+ * @version 0.0.1
+ */
+public class JPanelSearchAdvancedVideo extends JPanel{
```
Comment (Regis18) [inline src/main/java/com/jala/view/JPanelSearchAdvancedVideo.java:29]: Space between Jpanel and braces

File: src/main/java/com/jala/view/JPanelSearchVideo.java [hunk 9]
```diff
@@ -0,0 +1,163 @@
...
+    private JComboBox cmbFrameRate, cmbVideoCodec, cmbAudioCodec, cmbAspectRatio,
+            cmbDimension, cmbExtension;
+
+    /**
+     * The constructor method of the ViewConsole class to build an instance
+     */
+    public JPanelSearchVideo() {
+        setLayout(new GridBagLayout());
+        initComponent();
+
```
Comment (Regis18) [inline src/main/java/com/jala/view/JPanelSearchVideo.java:47]: Delete an extra space

**Pull request merged**